app/page/schedule/List/List.js

```diff
@@ -10,34 +10,73 @@ define([
 ], function (Page, can, content, Schedule) {
 	return Page.extend(
 		{
-
+			defaults: {
+				
+				template: content
+			}
 		},
 		{
-			template : content,
 
-			init: function (element) {
-                var isLoading = can.compute(true);
+			init: function (element, options) {
+
+				this.modelView = new can.Map(
+					{
+						schedule:null,
+						isLoading: true
+					}
+				);
+
+				element.html(options.template(this.modelView));
+
+				this.setSchedules();
+
+            },
 
-                var promise = Schedule.findAll({
+            setSchedules: function () {
```

Comment:
`setSchedules` funkcija treba da postavlja loading stanje na `true`, jer postoji poziv na nju iz then callback-a u `removeSchedule` funkciji. 

Takodje treba implementirati i removing stanje. Sad cu da dodam to u task.